### DISCUSSION TITLE
switching chains in dapps

### DISCUSSION THREAD

**Discussion by shortwavesurfer2009** [horizontalsystems/unstoppable-wallet-android#6272] [Q&A]
I am having trouble switching chains in dapps like aave and opensea. The request from walletconnectv2 sets ethereum and i cant switch it. When the dapp tries to switch chains it cant do it

**Comment by abdrasulov** [comment]
@shortwavesurfer2009 thanks for reporting. We'll check it soon

**Comment by Dianaaiym** [comment]
> I am having trouble switching chains in dapps like aave and opensea. The request from walletconnectv2 sets ethereum and i cant switch it. When the dapp tries to switch chains it cant do it

Hi,

Thank you for the report.
We added this task to the current sprint. However, it will be completed in the next update.
